src/RendererGL.cpp

```diff
@@ -519,7 +520,7 @@ namespace onut
                 programDirty ||
                 renderTargetDirty)
             {
-                // Because opengl sucks, we need to determine if we have to flip Y
+                // Because opengl sucks, we need to determine if we have to flip Y
```

Comment:
not sure what happened here... lol I think my IDE changed a tab to a white space.